apps/coscrad-frontend/src/store/slices/resources/shared/notes-for-resource/self-notes-panel.presenter.tsx

```diff
@@ -8,20 +7,12 @@ interface SelfNotesPanelProps {
     notes: SelfConnectionNote[];
 }
 
-export const SelfNotesPanelPresenter = ({
-    compositeIdentifier: { type: resourceType, id },
-    notes,
-}: SelfNotesPanelProps): JSX.Element => (
+export const SelfNotesPanelPresenter = ({ notes }: SelfNotesPanelProps): JSX.Element => (
     // TODO remove magic string
     <>
-        <Typography variant="h3">
-            {/* TODO Use standard formatter */}
-            Notes for {resourceType}/{id}
-        </Typography>
-        <Box data-testid="selfNotesPanel">
-            {notes.length > 0
-                ? notes.map((note) => <SelfNotePresenter {...note} key={note.id} />)
-                : 'No Notes Found'}
-        </Box>
+        <div data-testid="selfNotesPanel" />
```

Comment:
I'm not a big fan of this approach.  When I write a Cypress test I expect to be able to access the children of the element with this `data-testid` and make further assertions. Perhaps the answer is not to use this attribute in the Cypress test at all. You can cross that bridge as you work on the Cypress test.